scale/storage/brokers/host/host_broker.py

```diff
@@ -0,0 +1,99 @@
+"""Defines a broker that mounts a local host directory into the task container as its backend storage"""
```

Comment:
Why is this module in it's own package instead of just in /brokers like the others?